Fixes #55: Check commit comment length

Ichabond opened the pull request:
Raise an InvalidInputError when the commit message is longer than 60 characters, as this breaks IOS-XR.

Ichabond committed:
Fixes #55: Check commit comment length

Raise an InvalidInputError when the commit message is longer than 60 characters, as this breaks IOS-XR.

luke-orden commented on pyIOSXR/iosxr.py:
```diff
@@ -579,14 +579,17 @@ def commit_config(self, label=None, comment=None, confirmed=None):
+            if len(comment) <= 60:
+                rpc_command += ' Comment="%s"' % comment
+            else:
+                raise InvalidInputError('comment needs to be shorter than 60 characters', self)
```
It can be up to and including 60 characters

Ichabond committed:
Update error message for commit comment length error

Rephrase the commit comment error to better reflect the limits.

Ichabond commented:
Thanks to @loverend for the reworded message.

mirceaulinic commented on pyIOSXR/iosxr.py:
```diff
@@ -579,14 +579,17 @@ def commit_config(self, label=None, comment=None, confirmed=None):
+            if len(comment) <= 60:
+                rpc_command += ' Comment="%s"' % comment
+            else:
+                raise InvalidInputError('comment should be no more than 60 characters', self)
```
We should just trim the commit message when it's longer than 60 characters.

Ichabond committed:
Truncate comment length to maximum of 60

Instead of raising an exception, just truncate the comment length to the maximum of 60
